static/downloadables/runPrsCLI.sh

```diff
@@ -664,7 +666,7 @@ calculatePRS () {
         # saves them to files
         # associations --> either allAssociations.txt OR associations_{fileHash}.txt
         # clumps --> {superPop}_clumps_{refGen}.txt
-        if $pyVer $SCRIPT_DIR/connect_to_server.py "$refgen" "${traits}" "${studyTypes}" "${studyIDs}" "$ethnicities" "$superPop" "$fileHash" "$extension" "$defaultSex"; then
+        if $pyVer "$SCRIPT_DIR/connect_to_server.py" "$refgen" "${traits}" "${studyTypes}" "${studyIDs}" "$ethnicities" "$superPop" "$fileHash" "$extension" "$defaultSex"; then
```

Comment:
Did you test the code with the "" around the file names? I know it works without them, I just want to make sure the "" doesn't change anything

Reply:
The quotes ensure that file path names with spaces in them are preserved as single paths and not confused as multiple variables.